qiskit_experiments/calibration/calibrations.py

```diff
@@ -0,0 +1,798 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Class to store and manage the results of calibration experiments."""
+
+from collections import namedtuple, defaultdict
+from datetime import datetime
+from typing import Any, Dict, Set, Tuple, Union, List, Optional
+import csv
+import dataclasses
+import regex as re
+
+from qiskit.pulse import (
+    Schedule,
+    ScheduleBlock,
+    DriveChannel,
+    ControlChannel,
+    MeasureChannel,
+    Call,
+    Instruction,
+)
+from qiskit.pulse.channels import PulseChannel
+from qiskit.circuit import Parameter, ParameterExpression
+from qiskit_experiments.calibration.exceptions import CalibrationError
+from qiskit_experiments.calibration.parameter_value import ParameterValue
+
+ParameterKey = namedtuple("ParameterKey", ["schedule", "parameter", "qubits"])
```

Comment:
Could we make the order `["parameter", "qubits", "schedule"]`? That seems to be the order of parameters in most functions. Also, could we try to make the parameter order as consistent as possible in the functions in the module? The only other thing I see not consistent with this order now is `parameters_table`.

The main benefit I see to changing the order for `ParameterKey` is that then it matches the form expected for `free_params` in `get_schedule`. Then one can do `free_params=[key]` in `get_schedule` and `calibration_parameter(*key)`

Reply:
Sounds good. See here: d85f5f6